pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -1000,6 +1000,8 @@ protected boolean commitSegment(String controllerVipUrl, boolean isSplitCommit)
     SegmentCompletionProtocol.Response commitResponse = commit(controllerVipUrl, isSplitCommit);
 
     if (!commitResponse.getStatus().equals(SegmentCompletionProtocol.ControllerResponseStatus.COMMIT_SUCCESS)) {
+      _segmentLogger.warn("Controller response was {} and not {}", commitResponse.getStatus(),
```

Comment:
Please move this log to the place where the return value is detected as not being SUCCESS. We already have other logs for non-success cases in the layers below.